storage/innobase/xtrabackup/test/t/pxb-1679.sh

```diff
@@ -27,26 +27,17 @@ CREATE TABLE test02 (id int auto_increment primary key, a TEXT, b TEXT);
 INSERT INTO test02 SELECT * FROM test01;
 EOF
 
-xtrabackup --backup --lock-ddl=false --target-dir=$topdir/backup 2>&1 | tee /dev/stderr | \
-    while read line ; do
-        echo $line
-        if [ $( echo $line | grep -i -c './test/test01#p#p3.ibd') -eq 1 ]; then
-            mysql -e "ALTER TABLE test01 TRUNCATE PARTITION p3" test
-        fi
-        if [ $( echo $line | grep -c './test/test02.ibd') -eq 1 ]; then
-            mysql -e "TRUNCATE TABLE test02" test
-        fi
-    done
-
-if ! [ ${PIPESTATUS[0]} -eq 0 ] ; then
-    die "backup failed"
-fi
-
+xtrabackup --backup --target-dir=$topdir/backup  
```

Comment:
We got rid of lock-ddl=FALSE and this is part of reproducible test case of https://jira.percona.com/browse/PXB-1679

Reply:
yes, concurrent truncate is not a safe operation with lock-ddl=false. I saw [regression](https://pxb.cd.percona.com/job/percona-xtrabackup-8.0-test-param/256/CMAKE_BUILD_TYPE=RelWithDebInfo,DOCKER_OS=debian%3Abuster,XTRABACKUP_TARGET=innodb80/testReport/junit/(root)/t_pxb-1679/sh/)

Reply:
@rahulmalik87 what I mean is that https://github.com/percona/percona-xtrabackup/pull/573/files added code to handle this situation. Why is this broken now?

Reply:
I have dropped the changes and created [PXB-2948 ](https://jira.percona.com/browse/PXB-2948) for tracking.

Reply:
Ok, can you please start work on investigating it so it doesn't die on backlog?